xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
yes, it is injected for every scan at XPD so I didn't list it. I can add it if it's helpful